File: php/1672_Richest_Customer_Wealth.php
<?php

class Solution
{

    /**
     * @param Integer[][] $accounts
     * @return Integer
     */
    function maximumWealth($accounts)
    {
        $maxSum = 0;
        $tempSum = 0;
        foreach ($accounts as $arAccount) {
            foreach ($arAccount as $price) {
                $tempSum += $price;
            }
            $maxSum = max($maxSum, $tempSum);
            $tempSum = 0;
        }

        return $maxSum;
    }

}

File: cpp/1672_Richest_Customer_Wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int maxSum = 0;
        int tempSum = 0;
        for (vector<int> account : accounts) {
            for (int price : account) {
                cout << price;
                tempSum += price;
            }
            maxSum = max(maxSum, tempSum);
            tempSum = 0;
        }

        return maxSum;
    }
};